### DISCUSSION TITLE
List of digital implementation requirements for new Decade Actions

### DISCUSSION THREAD

**Discussion by pbuttigieg** [iodepo/OceanDecade-dsig#28] [Ideas]
In our DSIG meeting today, the proposal to require new Decade Actions to comply to a minimal set of implementation requirements. This would also be retrospective. 

How shall we shape this?

@Lucy-Scott @LDemargne @kevin-obrien

**Comment by pbuttigieg** [comment]
This is similar to the recommendations given by the Strategy Group to new Actions, requested by the DCU

**Reply by LDemargne** [reply]
Always remember to state the benefits, return on investment, credit people will get for sharing their data through existing data repositories/centres and thus contributing to the global data and information ecosystem.

**Reply by pbuttigieg** [reply]
Cross-link to crediting issues: https://github.com/iodepo/OceanDecade-dsig/discussions/26#discussioncomment-7072646
